addons/l10n_es/models/template_es_assec.py

```diff
@@ -9,7 +9,7 @@ class AccountChartTemplate(models.AbstractModel):
     @template('es_assec')
     def _get_es_assec_template_data(self):
         return {
-            'name': 'PGCE entidades sin ánimo de lucro 2008',
+            'name': _('Spanish CoA for non-profit entities (2008)'),
```

Comment:
"Spain" is already in the displayed name in the selection
```suggestion
            'name': _('Non-profit entities (2008)'),
```

Same for the other templates